fluent-package/Rakefile

```diff
@@ -116,7 +116,7 @@ end
 
 def template_params(params = nil)
   config = {
-    project_name: PACKAGE_NAME,
+    project_name: INTERNAL_PACKAGE_DIR,
```

Comment:
It seems that `INTERNAL_` prefix is no longer needed.
Simply `PACKAGE_DIR` seems better for this.

Reply:
okay, I'll fix it.